### PR TITLE
Wcms 1100 improving build process

### PR DESCRIPTION
**Pull request by DaRaFF**
As long as we are using the render decorator on node only, the build task (which caused some problems with grunt bump) is not used anymore.

When we reintroduce client side support, we should solve it completely else then now (after @marcbachmann's feedback)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [3.0.0] - 2015-03-17
```
Comment (kurakin-oleksandr) [inline CHANGELOG.md]: What is a correct version? 3.0.0 or 2.2.8?

Comment (DaRaFF) [inline CHANGELOG.md]: you can take 2.2.8

3.0.0 is, because i have removed a grunt task.

**Comment by kurakin-oleksandr** [comment]
So 3.0.0 or 2.2.8?
In your package.json version is 2.2.8 and in README 3.0.0. You should fix it

**Comment by kurakin-oleksandr** [comment]
Em, why did you merge it?

**Comment by DaRaFF** [comment]
As explained in the skype call, we create new tags and update packages in the master branch, like for alle other livingdocs projects.

I will update the readme to explain why it makes sense (do it the same way then all other livingdocs projects)

**Pull request merged**